CHANGELOG.rst

```diff
@@ -7,6 +7,7 @@ Version 3.2.0
 - Added meta classes for providers for internal usage (see `#621 <https://github.com/lk-geimfari/mimesis/issues/621>`_.)
 - Added support for custom templates in ``Person().username()``
 - Added ``ItalianSpecProvider()``
+- Added builtin provider in Generic.
```

Comment:
It's better to add more info "An implemented mechanism for automatically add a built-in provider for a current locale to Generic" or something like this.